packages/main/src/components/FilterGroupItem/index.tsx

```diff
@@ -86,7 +86,6 @@ FilterGroupItem.displayName = 'FilterGroupItem';
 FilterGroupItem.defaultProps = {
   groupName: 'default',
   visible: true,
-  visibleInFilterBar: true,
```

Comment:
Removing this default would result in a breaking change, wouldn't it?

Reply:
This is now handled internally by the following lines:
https://github.com/SAP/ui5-webcomponents-react/blob/6f7a75c30c11a187aab21d7c5d95fbbf7752565f/packages/main/src/components/FilterBar/index.tsx#L136-L138

The `FilterBar` needs to know when the `visibleInFilterBar` prop is set and when it is `undefined`. Otherwise toggling the filters inside the FilterDialog did overwrite the prop and therefore it wasn't possible to manually show/hide the filter inside the `FilterBar`. 

As the functionality doesn't change for  `FilterGroupItems` with or without the `visibleInFilterBar` prop, it's not a breaking change in my opinion.